src/Model/User.php

```diff
@@ -304,6 +304,33 @@ private static function updatePasswordHashed($uid, $pasword_hashed)
 		return dba::update('user', $fields, ['uid' => $uid]);
 	}
 
+	/**
+	 * @brief Checks if a nickname is in the list of the forbidden nicknames
+	 *
+	 * Check if a nickname is forbidden from registration on the node by the
+	 * admin. Forbidden nicknames (e.g. role namess) can be configured in the
+	 * admin panel.
+	 *
+	 * @param string $nickname The nickname that should be checked
+	 * @return boolean True is the nickname is blocked on the node
+	 */
+	public static function isNicknameBlocked($nickname)
+	{
+		$forbidden_nicknames = Config::get('system', 'forbidden_nicknames', '');
```

Comment:
Just realized that on existing nodes the admin has to once safe the site config to make this effective. Should I add the `$default_forbidden_nicknames` maybe to the class and use it here as well?

Reply:
Not necessarily, my PR #5295 will add a default value for this config key outside of the code itself, in the defaults config file itself. And depending on how quickly #5314 is merged, this could happen this week-end.